src/Exception/CompositeException.php

```diff
@@ -21,7 +21,7 @@ public function __construct(array $exceptions, $message = '', $code = 0, $previo
     }
 
     /**
-     * @return \Throwable[]|\Exception[]
+     * @return \Throwable[]
```

Comment:
LGTM locally, but makes me wonder if the method name `getExceptions()` is still applicable?

Reply:
Tbh it doesn't make sense so I've renamed it

Reply:
To be consistent, the property/variables could also be renamed.